spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -727,51 +740,97 @@ def write_sorting(sorting, save_path, **nwbfile_kwargs):
                     spkt = sorting.get_unit_spike_train(unit_id=id) / fs # spike trains withinin the SortingExtractor object are not scaled by sampling frequency
                     nwbfile.add_unit(id=id, spike_times=spkt)
 
+            # Always add column for shank ids if a MultiSortingExtractor
+            if isinstance(sorting,se.MultiSortingExtractor):
+                set_dynamic_table_property(
+                        dynamic_table=nwbfile.units,
+                        row_ids=[int(k) for k, v in sorting._unit_map.items()],
+                        property_name='shank_id',
```

Comment:
I would call this `group`, which is more general. For tetrodes, for example, you have multiple groups, but they are not on different shanks.

Reply:
I'm fixing this real quick :)